blockly/src/server/Server.java

```diff
@@ -1384,11 +1384,15 @@ public void moveToExit() {
     if (Game.currentLevel().exitTiles().isEmpty()) return;
     Tile exitTile = Game.currentLevel().exitTiles().getFirst();
 
+    PositionComponent pc =
+        hero.fetch(PositionComponent.class)
+            .orElseThrow(() -> MissingComponentException.build(hero, PositionComponent.class));
+
     GraphPath<Tile> pathToExit =
-        LevelUtils.calculatePath(EntityUtils.getHeroCoordinate(), exitTile.coordinate());
+        LevelUtils.calculatePath(pc.position().toCoordinate(), exitTile.coordinate());
 
     for (Tile nextTile : pathToExit) {
-      Tile currentTile = Game.tileAT(EntityUtils.getHeroCoordinate());
+      Tile currentTile = Game.tileAT(pc.position().toCoordinate());
```

Comment:
ich versteh den back change nicht

Reply:
ja, aber über `EntityUtils` sparr ich mir den ganzen fetch kram da oben.
Ich mein wenn wir schon ne Utils klasse haben, why dont use it?

Reply:
Asoo. Die Sache ist das in `EntityUtils` einfach auch nur gefetcht wird. Und hier haben wir eine Schleife, somit wäre es minimal performanter einfach nur einmal das Component zu hohlen (plus wir haben schon das Entity). Soll ich es wieder reverten, für die lesbarkeit?

Reply:
ne bin ich fine mit, ich habs nur nicht verstanden gehabt.

Ist ja so auch gut lesbar